docker-entrypoint.sh

```diff
@@ -273,6 +273,20 @@ if expr "$1" : "apache" 1>/dev/null || [ "$1" = "php-fpm" ] || [ "${NEXTCLOUD_UP
         fi
     ) 9> /var/www/html/nextcloud-init-sync.lock
 
+    # check if config files of user installation differ from image
+    for cfgPath in /usr/src/nextcloud/config/*.php; do
+        cfgFile=$(basename "$cfgPath")
+
+        if [ "$cfgFile" != "config.sample.php" ]; then
+            if ! cmp -s "/usr/src/nextcloud/config/$cfgFile" "/var/www/html/config/$cfgFile"; then
+                echo "Warning: your config folder is missing the file $cfgFile that comes with this Docker image or it differs from the original file which could cause certain Docker configuration environment variables to be ignored or not work as expected."
```

Comment:
```suggestion
                echo "***WARNING:*** Your config folder is missing the file $cfgFile that comes with this Docker image or it differs from the original file which could cause certain Docker configuration environment variables to be ignored or not work as expected."
```

Came from seeing issues like [this]( https://github.com/linuxserver/docker-nextcloud/issues/400) in the ls.io image where they have some similar warnings, but it being overlooked even though it's likely the cause of the reported issue (outdated nginx configs in this case) :-)